compiler/Makefile

```diff
@@ -182,7 +182,7 @@ frontend-shared: FORCE $(CHPL_CONFIG_CHECK) $(COMPILER_BUILD)
 
 test-frontend: FORCE frontend-tests $(CHPL_CONFIG_CHECK) $(COMPILER_BUILD)
 	@echo "Running the frontend tests..."
-	JOBSFLAG=`echo "$$MAKEFLAGS" | sed -n 's/.*\(-j\|--jobs=\) *\([0-9][0-9]*\).*/-j\2/p'` ; \
+	JOBSFLAG=`echo "$$MAKEFLAGS" | sed -n -E 's/.*(-j|--jobs=) *([0-9][0-9]*).*/-j\2/p'` ; \
```

Comment:
Should be this `-r` instead of `-E`? The `man` page for macOS sed says:

```
     -r      Same as -E for compatibility with GNU sed.
```

Which makes me think that that '-E' is the "custom" flag and '-r' is the compatibility flag. Am I wrong?

Reply:
man page for `sed` on Ubuntu says:

```
       -E, -r, --regexp-extended

              use extended regular expressions in the script (for  portability
              use POSIX -E).
```

So, I think `-E` is the right one to use here (to the extent the difference matters).